mllib/src/main/scala/org/apache/spark/mllib/clustering/LDA.scala

```diff
@@ -220,6 +206,38 @@ class LDA private (
     this
   }
 
+
+  /** LDAOptimizer used to perform the actual calculation */
+  def getOptimizer(): LDAOptimizer = ldaOptimizer
+
+  /**
+   * LDAOptimizer used to perform the actual calculation (default = EMLDAOptimizer)
+   */
+  def setOptimizer(optimizer: LDAOptimizer): this.type = {
+    this.ldaOptimizer = optimizer
+    this
+  }
+
+  /**
+   * Set the LDAOptimizer used to perform the actual calculation by algorithm name.
+   * Currently "EM" is supported.
```

Comment:
Could this be made case-insensitive?  Other algorithms mostly use lowercase string values.  We can note it in the doc too.